integrations/aws/utils/resources.py

```diff
@@ -222,9 +232,18 @@ async def resync_cloudcontrol(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
                         e.response.get("Error", {}).get("Code")
                         == "AccessDeniedException"
                     ):
+                        if is_global:
+                            logger.warning(
+                                f"Access denied in global resource {kind} in region {region}. Querying again in all regions"
+                            )
+                            async for batch in resync_cloudcontrol(
+                                kind, is_global=False
+                            ):
+                                yield batch
```

Comment:
add message about the why and the intention

Reply:
I did a comment explaining why, but then decided to leave it as as a log to inform the user as well, check out the log above, doesn't it suffice the "Why"?